jobs/riemann/templates/config/k8s.clj.erb

```diff
@@ -0,0 +1,17 @@
+(defn k8s-alerts [pd stable-time]
+  (where (host "riemann-podstatus")
+    (by :service
```

Comment:
Not super happy with using a by here, as the list of pods will grow over time, but I'm not sure how else to tackle this, and I think we restart riemann frequently enough via routine updates that this won't be an issue.